1-index.html

```diff
@@ -17,7 +17,7 @@
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
       <h2>What is Jamulus?</h2>
-      Jamulus is software for playing music, rehearsing, or just jamming with anyone online with low latency. You can use your Windows, macOS or Linux machine to connect to Jamulus servers worldwide. Jamulus is free and you can just use your normal broadband connection. Simply connect to a public server or host your own private one. Jamulus has been in development since 2006 and is designed for high quality, low-latency sound, making it easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank" rel="noreferrer">how people do it</a>!
+       Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. With high quality, low-latency sound on a normal broadband connection, it's easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank" rel="noreferrer">how people do it</a> - and  <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">download it here</a>!
```

Comment:
Sounds good. However we‘re not linking to the download page here? So wouldn’t it be better to say get started here?

Reply:
I thought we'd link to that until we had the download page. It's got the download buttons on it.

Reply:
Ok. So happy to approve.